# HG changeset patch
# Parent 87b7e40e889927d8c32f43783a7dcf100d022118
Quieten C-state messages for setting c-state information for non-existent physical processors.

diff -r 87b7e40e8899 xen/arch/x86/acpi/cpu_idle.c
--- a/xen/arch/x86/acpi/cpu_idle.c
+++ b/xen/arch/x86/acpi/cpu_idle.c
@@ -1038,7 +1038,7 @@ long set_cx_pminfo(uint32_t cpu, struct 
     cpu_id = get_cpu_id(cpu);
     if ( cpu_id == -1 )
     {
-        printk(XENLOG_ERR "no cpu_id for acpi_id %d\n", cpu);
+        printk(XENLOG_INFO "no cpu_id for acpi_id %d\n", cpu);
         return -EINVAL;
     }
 
